repository/q.json

```
@@ -252,6 +252,18 @@
 				}
 			]
 		},
+		{
+			"name": "QuickLineNavigator",
+			"author": "ChenZhu-Xie",
+			"details": "https://github.com/ChenZhu-Xie/QuickLineNavigator",
+			"labels": ["search", "fuzzy search", "accurate search", "pre-filter", "color", "highlight", "highlighter", "grep", "ugrep"],
```

Several of these labels are unique to this package. They won't help users find your package (ie. it won't improve your "SEO" inside package control) and are better kept in the description. I would propose reducing it down to:
```suggestion
			"labels": ["search", "grep", "highlight"],
```